CHANGES.rst

```diff
@@ -1,13 +1,20 @@
 .. currentmodule:: dirty_cat
 
+Release 0.4.0
+==============
+
+* New feature: joining tables using :func:`fuzzy_join` by approximate key matching. Matches are based
+  on string similarities and the nearest neighbors matches are found for each category. 
+* **datasets.fetching**: contains a new function :func:`fetch_world_bank_indicator` that can be
+    used to download any indicator from the World Bank Open Data platform. It only needs the 
+    indicator ID that can be found on the website.
+
```

Comment:
Two comments on that: could you please fix the links and apply the title convention we usually use ("Major changes" / "Minor changes" / "Bug-fixes" / "Notes")?